README.md

```diff
@@ -418,6 +414,25 @@ By default unleash-client fetches the feature toggles from unleash-server every
 * When .json file does not exists
 * When the named feature toggle does not exist in .json file
 
+The backup file name will follow this pattern: `{fileNameWithoutExtension}-{AppName}-{InstanceTag}-{SdkVersion}{extension}`, where InstanceTag is either what you configure on `UnleashSettings` during startup, or a formatted string with a random component following this pattern: `{Dns.GetHostName()}-generated-{Guid.NewGuid()}`.
```

Comment:
I'm guessing?

```suggestion
The backup file name will follow this pattern: `{fileNameWithoutExtension}-{AppName}-{InstanceTag}-{SdkVersion}.{extension}`, where InstanceTag is either what you configure on `UnleashSettings` during startup, or a formatted string with a random component following this pattern: `{Dns.GetHostName()}-generated-{Guid.NewGuid()}`.
```

Reply:
Interesting one. I lifted this from the code, but it makes more sense docs wise to have the period in